api-docs/api-operations/methods/post-restart-ha-instance-version-accountid-ha-haid-action.rst

```diff
@@ -0,0 +1,148 @@
+
+.. _post-restart-ha-instance-version-accountid-ha-haid-action.rst:
+
+Restart a HA Instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code::
+
+    POST /{version}/{accountId}/ha/{haId}/action
+
+Restarts a HA Instance. When the HA instance is being restarted, it goes into a ``REBOOT`` state. It switches back to ``ACTIVE`` once the operation is complete.
+
```

Comment:
Restarts an HA instance. During the restart process, the HA Instance goes into a ``REBOOT`` state. After the operation completes, the status changes  to ``ACTIVE``.